### ISSUE TITLE
Pre-work Checklist: Developer: Christina Or

### ISSUE THREAD

**Issue by christinaor** [hackforla/website#3827]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by HackforLABot** [comment]
## Activity Log: christinaor
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- christinaor opened: https://github.com/hackforla/website/issues/3827 at 01/16/2023, 07:47 PM PST
- christinaor assigned: https://github.com/hackforla/website/issues/3827 at 01/16/2023, 08:04 PM PST
- christinaor assigned: https://github.com/hackforla/website/issues/2435#issuecomment-1385988822 at 01/29/2023, 03:43 PM PST
- christinaor opened: https://github.com/hackforla/website/pull/3894 at 01/29/2023, 04:58 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/2435#issuecomment-1407838074 at 01/29/2023, 05:05 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/2435#issuecomment-1407842166 at 01/29/2023, 05:09 PM PST
- christinaor PR was merged: https://github.com/hackforla/website/pull/3894#event-8402658890 at 01/31/2023, 10:33 AM PST
- christinaor issue closed by PR 3894: https://github.com/hackforla/website/issues/2435#event-8402659117 at 01/31/2023, 10:33 AM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3883#issuecomment-1411410797 at 01/31/2023, 07:27 PM PST
- christinaor submitted review: https://github.com/hackforla/website/pull/3883#pullrequestreview-1278387817 at 01/31/2023, 07:36 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3901#issuecomment-1411419894 at 01/31/2023, 07:45 PM PST
- christinaor submitted review: https://github.com/hackforla/website/pull/3901#pullrequestreview-1278450280 at 01/31/2023, 09:25 PM PST
- christinaor assigned: https://github.com/hackforla/website/issues/3360#issuecomment-1192033274 at 01/31/2023, 09:28 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/3360#issuecomment-1411490485 at 01/31/2023, 09:30 PM PST
- christinaor opened: https://github.com/hackforla/website/pull/3926 at 02/02/2023, 01:46 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/3360#issuecomment-1414423314 at 02/02/2023, 01:51 PM PST
- christinaor closed as completed: https://github.com/hackforla/website/issues/3827#event-8425904716 at 02/02/2023, 01:54 PM PST
- christinaor assigned: https://github.com/hackforla/website/issues/3572#issuecomment-1407846561 at 02/02/2023, 02:30 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/3572#issuecomment-1414463005 at 02/02/2023, 02:32 PM PST
- christinaor opened: https://github.com/hackforla/website/pull/3928 at 02/02/2023, 08:02 PM PST
- christinaor PR was merged: https://github.com/hackforla/website/pull/3926#event-8452546471 at 02/06/2023, 02:53 PM PST
- christinaor issue closed by PR 3926: https://github.com/hackforla/website/issues/3360#event-8452546788 at 02/06/2023, 02:53 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3928#issuecomment-1420061818 at 02/06/2023, 06:02 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3928#issuecomment-1420068613 at 02/06/2023, 06:05 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3928#issuecomment-1420171507 at 02/06/2023, 08:09 PM PST
- christinaor PR was merged: https://github.com/hackforla/website/pull/3928#event-8465775348 at 02/07/2023, 11:53 PM PST
- christinaor issue closed by PR 3928: https://github.com/hackforla/website/issues/3572#event-8465775584 at 02/07/2023, 11:53 PM PST
- christinaor assigned: https://github.com/hackforla/website/issues/3546#issuecomment-1304901119 at 02/10/2023, 04:12 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/3546#issuecomment-1426517805 at 02/10/2023, 04:15 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/3546#issuecomment-1432116661 at 02/15/2023, 02:14 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3979#issuecomment-1433498511 at 02/16/2023, 10:03 AM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3979#issuecomment-1433701720 at 02/16/2023, 12:54 PM PST
- christinaor submitted review: https://github.com/hackforla/website/pull/3979#pullrequestreview-1302313156 at 02/16/2023, 01:36 PM PST
- christinaor opened: https://github.com/hackforla/website/pull/3994 at 02/16/2023, 07:29 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/3546#issuecomment-1434057796 at 02/16/2023, 07:50 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3984#issuecomment-1435065406 at 02/17/2023, 10:21 AM PST
- christinaor submitted review: https://github.com/hackforla/website/pull/3984#pullrequestreview-1304192572 at 02/17/2023, 12:58 PM PST
- christinaor submitted review: https://github.com/hackforla/website/pull/4002#pullrequestreview-1304270116 at 02/17/2023, 02:21 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3994#issuecomment-1437717016 at 02/20/2023, 04:58 PM PST
- christinaor PR was merged: https://github.com/hackforla/website/pull/3994#event-8565955177 at 02/20/2023, 05:15 PM PST
- christinaor issue closed by PR 3994: https://github.com/hackforla/website/issues/3546#event-8565955289 at 02/20/2023, 05:15 PM PST
- christinaor assigned: https://github.com/hackforla/website/issues/3976 at 02/20/2023, 06:22 PM PST
- christinaor commented on: https://github.com/hackforla/website/issues/3976#issuecomment-1437779328 at 02/20/2023, 06:37 PM PST
- christinaor opened: https://github.com/hackforla/website/pull/4018 at 02/20/2023, 08:42 PM PST
- christinaor PR was merged: https://github.com/hackforla/website/pull/4018#event-8598812544 at 02/23/2023, 11:06 PM PST
- christinaor issue closed by PR 4018: https://github.com/hackforla/website/issues/3976#event-8598812767 at 02/23/2023, 11:06 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/3979#issuecomment-1444152258 at 02/24/2023, 10:00 AM PST
- christinaor submitted review: https://github.com/hackforla/website/pull/3979#pullrequestreview-1314539723 at 02/25/2023, 08:29 AM PST
- christinaor opened: https://github.com/hackforla/website/issues/4101 at 03/03/2023, 09:56 AM PST
- christinaor issue closed by PR 3928: https://github.com/hackforla/website/issues/3572#event-8691600687 at 03/07/2023, 03:54 PM PST
- christinaor commented on: https://github.com/hackforla/website/pull/4178#issuecomment-1470398978 at 03/15/2023, 09:52 AM PDT
- christinaor submitted review: https://github.com/hackforla/website/pull/4178#pullrequestreview-1342079628 at 03/15/2023, 10:53 AM PDT
- christinaor assigned: https://github.com/hackforla/website/issues/3959#issuecomment-1477059261 at 03/21/2023, 05:32 PM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1478827612 at 03/21/2023, 07:13 PM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1488922411 at 03/29/2023, 09:26 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1491019609 at 03/30/2023, 03:01 PM PDT
- christinaor commented on: https://github.com/hackforla/website/pull/4346#issuecomment-1493535080 at 04/02/2023, 07:06 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/4346#event-8907762974 at 04/02/2023, 07:10 PM PDT
- christinaor closed by PR 4346: https://github.com/hackforla/website/issues/2830#event-8907763110 at 04/02/2023, 07:10 PM PDT
- christinaor unassigned: https://github.com/hackforla/website/issues/3959#issuecomment-1496715092 at 04/04/2023, 04:19 PM PDT
- christinaor assigned: https://github.com/hackforla/website/issues/4357 at 04/06/2023, 10:06 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4357#issuecomment-1499484764 at 04/06/2023, 11:54 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4357#issuecomment-1502731125 at 04/10/2023, 11:01 PM PDT
- christinaor opened: https://github.com/hackforla/website/pull/4550 at 04/20/2023, 10:17 PM PDT
- christinaor assigned: https://github.com/hackforla/website/issues/4277 at 04/21/2023, 01:43 PM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4277#issuecomment-1518348358 at 04/21/2023, 02:24 PM PDT
- christinaor opened: https://github.com/hackforla/website/issues/4555 at 04/21/2023, 02:57 PM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4277#issuecomment-1518372927 at 04/21/2023, 03:02 PM PDT
- christinaor PR was merged: https://github.com/hackforla/website/pull/4550#event-9171031612 at 05/04/2023, 12:50 PM PDT
- christinaor issue closed by PR 4550: https://github.com/hackforla/website/issues/4357#event-9171031818 at 05/04/2023, 12:50 PM PDT
- christinaor assigned: https://github.com/hackforla/website/issues/4563 at 05/10/2023, 08:45 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4563#issuecomment-1542438412 at 05/10/2023, 08:50 AM PDT
- christinaor opened: https://github.com/hackforla/website/issues/4647 at 05/10/2023, 09:46 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4563#issuecomment-1542522615 at 05/10/2023, 09:52 AM PDT
- christinaor assigned: https://github.com/hackforla/website/issues/4562 at 05/10/2023, 09:57 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4562#issuecomment-1542529473 at 05/10/2023, 09:58 AM PDT
- christinaor opened: https://github.com/hackforla/website/issues/4648 at 05/10/2023, 10:20 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4648#issuecomment-1542560264 at 05/10/2023, 10:25 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4647#issuecomment-1542567331 at 05/10/2023, 10:30 AM PDT
- christinaor commented on: https://github.com/hackforla/website/issues/4648#issuecomment-1548263062 at 05/15/2023, 10:28 AM PDT
- christinaor issue closed as completed: https://github.com/hackforla/website/issues/4563#event-9490654208 at 06/10/2023, 04:32 AM PDT
- christinaor submitted review: https://github.com/hackforla/website/pull/5161#pullrequestreview-1568283271 at 08/08/2023, 02:40 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5161#event-10041971345 at 08/08/2023, 08:33 PM PDT
- christinaor closed by PR 5161: https://github.com/hackforla/website/issues/5137#event-10041971414 at 08/08/2023, 08:33 PM PDT
- christinaor commented on: https://github.com/hackforla/website/pull/5152#issuecomment-1670611355 at 08/08/2023, 08:50 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5152#event-10042037090 at 08/08/2023, 08:50 PM PDT
- christinaor closed by PR 5152: https://github.com/hackforla/website/issues/5131#event-10042037162 at 08/08/2023, 08:50 PM PDT
- christinaor commented on: https://github.com/hackforla/website/pull/5275#issuecomment-1688982188 at 08/22/2023, 02:55 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5275#event-10163488667 at 08/22/2023, 02:55 PM PDT
- christinaor closed by PR 5275: https://github.com/hackforla/website/issues/5266#event-10163488790 at 08/22/2023, 02:55 PM PDT
- christinaor commented on: https://github.com/hackforla/website/pull/5261#issuecomment-1688986738 at 08/22/2023, 03:00 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5261#event-10163516063 at 08/22/2023, 03:00 PM PDT
- christinaor closed by PR 5261: https://github.com/hackforla/website/issues/5198#event-10163516215 at 08/22/2023, 03:00 PM PDT
- christinaor commented on: https://github.com/hackforla/website/pull/5259#issuecomment-1689006815 at 08/22/2023, 03:24 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5259#event-10163645997 at 08/22/2023, 03:24 PM PDT
- christinaor closed by PR 5259: https://github.com/hackforla/website/issues/5255#event-10163646147 at 08/22/2023, 03:24 PM PDT
- christinaor commented on: https://github.com/hackforla/website/pull/5335#issuecomment-1698380284 at 08/29/2023, 07:01 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5335#event-10227505144 at 08/29/2023, 07:01 PM PDT
- christinaor closed by PR 5335: https://github.com/hackforla/website/issues/5281#event-10227505231 at 08/29/2023, 07:01 PM PDT
- christinaor commented on: https://github.com/hackforla/website/pull/5331#issuecomment-1698382978 at 08/29/2023, 07:05 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5331#event-10227523028 at 08/29/2023, 07:05 PM PDT
- christinaor closed by PR 5331: https://github.com/hackforla/website/issues/5154#event-10227523109 at 08/29/2023, 07:05 PM PDT
- christinaor commented on: https://github.com/hackforla/website/pull/5296#issuecomment-1698387486 at 08/29/2023, 07:12 PM PDT
- christinaor submitted review: https://github.com/hackforla/website/pull/5296#pullrequestreview-1601756711 at 08/29/2023, 07:13 PM PDT
- christinaor submitted review: https://github.com/hackforla/website/pull/5296#pullrequestreview-1601757119 at 08/29/2023, 07:13 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5296#event-10227569539 at 08/29/2023, 07:16 PM PDT
- christinaor closed by PR 5296: https://github.com/hackforla/website/issues/5253#event-10227569633 at 08/29/2023, 07:16 PM PDT
- christinaor submitted review: https://github.com/hackforla/website/pull/5593#pullrequestreview-1645431457 at 09/26/2023, 06:27 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5593#event-10481500243 at 09/26/2023, 06:27 PM PDT
- christinaor closed by PR 5593: https://github.com/hackforla/website/issues/5329#event-10481500310 at 09/26/2023, 06:27 PM PDT
- christinaor submitted review: https://github.com/hackforla/website/pull/5586#pullrequestreview-1645437724 at 09/26/2023, 06:36 PM PDT
- christinaor merged: https://github.com/hackforla/website/pull/5586#event-10481543146 at 09/26/2023, 06:36 PM PDT
- christinaor closed by PR 5586: https://github.com/hackforla/website/issues/4961#event-10481543243 at 09/26/2023, 06:36 PM PDT
- christinaor submitted review: https://github.com/hackforla/website/pull/5570#pullrequestreview-1645482653 at 09/26/2023, 07:21 PM PDT
- christinaor issue closed as completed: https://github.com/hackforla/website/issues/4562#event-10597696534 at 10/09/2023, 06:56 PM PDT
- christinaor issue closed as completed: https://github.com/hackforla/website/issues/4563#event-11831724924 at 02/16/2024, 09:35 AM PST
- christinaor unassigned: https://github.com/hackforla/website/issues/4277 at 03/26/2024, 03:36 PM PDT
